bulk/repo-access-api.adoc

```diff
@@ -68,6 +68,13 @@ NOTE: We might add advanced filtering capabilities in the future, or introduce a
 
 If the repository supports <<trash>>, it MUST retrieve nodes from _trash_ if the client asks for them.
 
+[horizontal]
```

Comment:
This came up during the last discussion (although not with that name). The idea was that _deleting_ a node should be an explicit action, not merely implicitly by not mentioning it anywhere. Thus we need a place to keep such nodes.